deploy/helm/kuberhealthy/templates/khcheck-network-connection.yaml

```diff
@@ -26,7 +26,19 @@ spec:
 {{- end }}
       resources:
         requests:
-          memory: 5Mi
+          {{- if .Values.check.networkConnection.resources.requests.cpu }}
+          cpu: {{ .Values.check.networkConnection.resources.requests.cpu }}
+          {{- end }}
```

Comment:
this is the only resource.requests that get checked if set or not? seems like we always want to have requests values required but not necessarily limits so we don't need this if check?

Reply:
@joshulyne to be honest, I looked into existing hard-coded values for resource requests/limits and added the conditions as per that. It turns out that this is the only check without cpu requests as in https://github.com/Comcast/kuberhealthy/blob/master/deploy/helm/kuberhealthy/templates/khcheck-network-connection.yaml#L28 not sure why. Hence I added that condition only for this check.

Reply:
i agree with you, this is something we should have required when we added this check. i'm thinking setting `cpu: 10m` would be okay. would you mind adding that to this pr and removing the if statement?

Reply:
Sure thing , updated :)